app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/EmailController.scala

```diff
@@ -82,14 +84,18 @@ class EmailController @Inject() (
   ): Future[Result] =
```

Comment:
Logic
If either there is no cache for the user (old behaviour) or the email is None (new behaviour) then direct user to "what is your email" page